### DISCUSSION TITLE
Means of fully applying ROIs from one video to another

### DISCUSSION THREAD

**Discussion by cgsol** [flatironinstitute/CaImAn#1349] [Q&A]
Hello!

I'm very new to using CaImAn, so I apologize if this question is a bit trivial. I'm currently working with some 1p data, adapting from the demo pipeline and things have been working great. My main question is if there is some way to take the ROIs found by running CaImAn on one video and apply it to a different video of the exact same area.

I would like to do a comparison between regular/blocked activity, but am finding that with the blocked activity video, it is finding relatively different ROIs, making neuron to neuron matching between videos difficult. I've used the register_ROIs/register_multisession function and have had some success, but due to considerable differences in the found ROIs, am getting very little ROI intersection between videos. I see that you can run a seeded CNMF -- is this what I should look into further?

Thank you so much!

**Comment by cgsol** [comment]
While using CNMF-E, is there a means to provide component centers as seeds (as in #865 )? When using a NumPy array of component centers (attained via `cm.base.rois.com` and used similarly as in #865 ) as the input for `seed_method`, along with `init_iter = 1`, similar results as to #1130 appear with very few total components, and barely (if any) accepted components.

Thank you in advance!

**Reply by pgunn** [reply]
@cgsol Hello,
Sorry for the late response; I believe for the seeds you want to provide a binary mask rather than just the centers - at least this is how it works with CNMF initialisation (see demo_seeded_CNMF) and I assume it works the same way with CNMFE. Give it a try and let me know if this helps.
